ddtrace/tracer.py

```diff
@@ -554,7 +553,7 @@ def _generate_diagnostic_logs(self):
                 self._log_compat(logging.WARNING, "- DATADOG TRACER DIAGNOSTIC - %s" % msg)
             else:
                 if log.isEnabledFor(logging.INFO):
-                    msg = "- DATADOG TRACER CONFIGURATION - %s" % json.dumps(info)
+                    msg = "- DATADOG TRACER CONFIGURATION - %s" % info
```

Comment:
how confident are we that this won't leak any data that should not be sent to logs?

Reply:
No idea.